pkg/instancegroups/instancegroups.go

```diff
@@ -248,6 +248,20 @@ func (r *RollingUpdateInstanceGroup) validateClusterWithDuration(rollingUpdateDa
 	}
 }
 
+// validateClusterWithDurationLoop validates cluster 3 times to make sure cluster does not flap
+func (r *RollingUpdateInstanceGroup) validateClusterWithDurationLoop(rollingUpdateData *RollingUpdateCluster, duration time.Duration) error {
+	var err error
+	for i := 1; i <= 3; i++ {
+		err = r.validateClusterWithDuration(rollingUpdateData, duration)
+		if err != nil {
+			return err
+		}
+		time.Sleep(rollingUpdateData.LoopDuration)
```

Comment:
```suggestion
		time.Sleep(rollingUpdateData.ValidateSuccessDuration / 2)
```

Instead of specifying the duration between checks (applicable after a successful validation), I think it would be more understandable for the user to specify the duration over which validation must remain successful. The number of these checks (3 currently proposed) is more of an implementation detail.